app/utils/estimate_prioritization/prioritization_criteria.py

```diff
@@ -7,8 +7,8 @@
     ],
     'adjustment_testadj': [
         lambda estimate: (estimate['pop_adj'] is True) and (estimate['test_adj'] is True),
-        lambda estimate: (pd.isna(estimate['pop_adj'])) and (estimate['test_adj'] is True),
         lambda estimate: (estimate['pop_adj'] is True) and (pd.isna(estimate['test_adj'])),
+        lambda estimate: (pd.isna(estimate['pop_adj'])) and (estimate['test_adj'] is True),
```

Comment:
nit: try to avoid changes that just move stuff around but don't change the actual code

Reply:
This change is actually functional @annoyingpr. The estimate prioritization logic goes thru the prio criteria defined in this array one by one and applies them on a DF of estimates. The order in which the prioritization criteria are defined is important - thus the change in ordering is actually a functional change.

Reply:
oh i see now! good to know, my bad @123chrisc 😅